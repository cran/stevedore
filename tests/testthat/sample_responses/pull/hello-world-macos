{"status":"Pulling from library/hello-world","id":"latest"}
{"status":"Digest: sha256:be0cd392e45be79ffeffa6b05338b98ebb16c87b255f48e297ec7f98e123905c"}
{"status":"Pulling from library/hello-world","id":"linux"}
{"status":"Digest: sha256:27f70fe1ab54eebd2569a33f0b86e5db42b8ebda65d0647d8eceb87d34462efd"}
{"status":"Pulling from library/hello-world","id":"nanoserver-1709"}
{"errorDetail":{"message":"no matching manifest for linux/amd64 in the manifest list entries"},"error":"no matching manifest for linux/amd64 in the manifest list entries"}
