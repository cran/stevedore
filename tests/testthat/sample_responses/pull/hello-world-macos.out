Pulling from library/hello-world latest
Digest: sha256:be0cd392e45be79ffeffa6b05338b98ebb16c87b255f48e297ec7f98e123905c
Pulling from library/hello-world linux
Digest: sha256:27f70fe1ab54eebd2569a33f0b86e5db42b8ebda65d0647d8eceb87d34462efd
Pulling from library/hello-world nanoserver-1709
no matching manifest for linux/amd64 in the manifest list entries
